src/workerd/api/basics.c++

```diff
@@ -681,10 +787,25 @@ void AbortSignal::triggerAbort(
       }
     }
   } else {
-    reason = js.exceptionToJsValue(kj::mv(exception));
+    reason = js.exceptionToJsValue(kj::cp(exception));
   }
+
   canceler->cancel(kj::cp(exception));
```

Comment:
If I'm reading this right, the original code was trying to use a moved-from value at this point

Reply:
That does appear to be the case, yes. Which in practice probably meant its description appeared empty, if the exception ended up being reported anywhere...

Reply:
That said, since this line _is_ the last use of `exception`, you could use `kj::mv` on this line.